action.yaml

```diff
@@ -80,7 +83,7 @@ outputs:
     description: 'Name of workflow triggered'
 runs:
   using: 'docker'
-  image: 'docker://ghcr.io/splunk/wfe-test-runner-action/wfe-test-runner-action:v5.0.1'
+  image: 'Dockerfile'
```

Comment:
We're going to update this to new version when released, right?

Reply:
automation should handle that. please refer [releaserc](https://github.com/splunk/wfe-test-runner-action/blob/main/.releaserc)

Reply:
yup, the idea here is to use `Dockerfile` to be able to test the change and then it's fine to leave it there because the automation will override it when making a release as Darshan pointed out